File: app/typescript/src/puyoimage.ts
import { Config } from './config'

// ぷよの画像を管理するクラス
export class PuyoImage {
  private _config: Config

  constructor(config: Config) {
    this._config = config
  }

  // ぷよの画像を読み込む（実装は後で）
  load(): void {
    // TODO: 画像読み込み処理
  }
}

File: app/python/lib/puyoimage.py
"""ぷよ画像管理クラス"""

from lib.config import Config


class PuyoImage:
    """ぷよの画像を管理するクラス"""

    def __init__(self, config: Config) -> None:
        """ぷよ画像の初期化

        Args:
            config: ゲーム設定
        """
        self.config = config
